src/FondOfSpryker/Yves/EnhancedEcommerce/EnhancedEcommerceDependencyProvider.php

```diff
@@ -7,13 +7,86 @@
 
 class EnhancedEcommerceDependencyProvider extends AbstractBundleDependencyProvider
 {
+    public const ENHNACED_ECOMMERCE_DATALAYER_EXPANDER_PLUGINS = 'ENHNACED_ECOMMERCE_DATALAYER_EXPANDER_PLUGINS';
+    public const ENHNACED_ECOMMERCE_TWIG_PARAMETER_BAG_EXPANDER_PLUGINS = 'ENHNACED_ECOMMERCE_TWIG_PARAMETER_BAG_EXPANDER_PLUGINS';
+    public const ENHNACED_ECOMMERCE_CONTROLLER_EVENT_HANDLER = 'ENHNACED_ECOMMERCE_CONTROLLER_EVENT_HANDLER';
+
```

Comment:
Namespacing container container constants.. `FOS_...`

Reply:
FOS?

Reply:
fond of spryker